### DISCUSSION TITLE
result in `performance point count` is different from `performance breakdown`

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#27798] [Q&A] [answered]
[compressed-logs.zip](https://github.com/ppy/osu/files/14887200/compressed-logs.zip)

https://github.com/ppy/osu/assets/34775378/59af2cc7-0465-4599-a1ab-29664c46e81b

see the value of `performance point count` just after the song ends and the value in `performance breakdown`

141pp -> 128pp

[The replay that caused this question.zip](https://github.com/ppy/osu/files/14887238/The.replay.that.caused.this.question.zip)
This replay may not play back normally as expected.

version: 2024.312.1

**Comment by Walavouchey** [comment]
can reproduce with the provided replay, though it gave two different numbers on my machine (81 -> 74) than shown above

@ppy/team-client

**Reply by cdwcgt** [reply]
The different numbers (between replay and video) are due to the inconsistency of ctb playback. however, this problem has been tracked in [another issue thread](https://github.com/ppy/osu/issues/26436).

**Reply by Walavouchey** [reply]
neither issues seem to occur in other rulesets

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/pull/27808
